eventmesh-runtime/src/test/java/org/apache/eventmesh/runtime/client/impl/PubClientImpl.java

```diff
@@ -73,7 +73,7 @@ public void close() {
             task.cancel(false);
             super.close();
         } catch (Exception e) {
-            e.printStackTrace();
+            log.error("Error  occured during close operation",e);
         }
```

Comment:
Please refer to the format of the example I provided in the issue for recording the log:

![image](https://github.com/apache/eventmesh/assets/41445332/579c068b-a3bf-44b0-871a-dc5f52743201)

Regarding the 4 pull requests you submitted for issue #4733, we will only merge one. Please consolidate the changes into a single pull request.

Reply:
@Pil0tXia  sure    i will change to the format mentioned by you.